source/common/runtime/runtime_features.cc

```diff
@@ -118,6 +118,8 @@ constexpr const char* disabled_runtime_features[] = {
     "envoy.reloadable_features.remove_legacy_json",
     // Sentinel and test flag.
     "envoy.reloadable_features.test_feature_false",
+    // TODO(derekargueta) flip to true after a release cycle.
```

Comment:
Any reason to wait?  I don't think this is unacceptably high risk.
(please move above the sentinel flag if you disagree =P)

Reply:
No reason, was just erring on the cautious side but just noticed the comment `Most features should be enabled by default.` a couple lines above `disabled_runtime_features` 🙂